docs/guides/runtime_tracing.md

```diff
@@ -0,0 +1,27 @@
+# Runtime Tracing
+
+The Crystal runtime has a tracing feature for low level functionality. It prints diagnostic info about runtime internals.
+
+A program must be built with the flag `-Dtracing` to support tracing.
+At runtime, the individual tracing components can be enabled via the environment variable `CRYSTAL_TRACE`. It receives a comma separated list of sections to enable.
+
+- `CRYSTAL_TRACE=none` Disable tracing (default)
+- `CRYSTAL_TRACE=gc`: Enable tracing for the garbage collector
+- `CRYSTAL_TRACE=sched`: Enalbe tracing for the scheduler
+- `CRYSTAL_TRACE=gc,sched`: Enable tracing for the garbage collector and scheduler
+- `CRYSTAL_TRACE=all` Enable all tracing
```

Comment:
Is this true?

```suggestion
- `CRYSTAL_TRACE=all` Enable all tracing (equivalent to `gc,sched`)
```

Reply:
Yes, but only until we add more sections.

Reply:
I suppose we can add this explanation for now and reconsider when we update for additional sections. I figure we won't want to spell out all combinations of them as we do now with only 2 options.